cli/commands/build.js

```diff
@@ -31,7 +31,16 @@ function getDestinationFromConfig(source, target) {
     throw new Error("clio.toml was not found on the specified directory.");
   }
 
-  const buildConfig = packageConfig.getPackageConfig().build;
+  const config = packageConfig.getPackageConfig();
+  const buildConfig = config.build;
+
+  // TODO: Clean up assignment
```

Comment:
```javascript
const buildTarget = target || buildConfig.build in config.target ?
  config.target[buildConfig.build].target : buildConfig.target
```
or

```javascript
const { target: configTarget } = config.target[buildConfig.build] || {}
const buildTarget = target || buildConfigTarget || buildConfig.target
```
we should find a clean way to replace this try / catch

Reply:
Ah, thanks for the suggestions!

Reply:
They both do not work, error is still thrown

Reply:
Righto, didn't test them, I'll try to find a way. I mean, one of those should work, with some modifications.

Reply:
oops, I had some `undefined` variables in my code, I edited it XD

Reply:
Fixed with a00a050